model/clusters_mgmt/v1/cluster_image_registry_type.model

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright (c) 2025 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// ClusterImageRegistry defines the image registry capability for an OpenShift cluster.
```

Comment:
We don't need to talk about the "capability" concept as it is an OCP implementation detail